src/value_generators/src/DocumentGenerator.cpp

```diff
@@ -1797,7 +1902,8 @@ Out valueGenerator(const Node& node,
  * If adding a new parsers map, remember to insert its entries into `allParsers` at the end of this
  * lambda function.
  */
-const auto [allParsers, arrayParsers, dateParsers, doubleParsers, intParsers, stringParsers] = [] () {
+const auto [allParsers, arrayParsers, dateParsers, doubleParsers, intParsers,
```

Comment:
Should we explicitly rename intParsers to int64Parsers now to go with the int32Parsers? Or at least add a comment explaining?